docs/source/howto/data.rst

```diff
@@ -805,3 +805,66 @@ This command will delete both the file repository and the database.
 .. danger::
 
   It is not possible to restore a deleted profile unless it was previously backed up!
+
+
+Transfering data
+================
+
+.. danger::
+
+    This feature is still in beta version and its API might change in the near future.
+    It is therefore not recommended that you rely on it for your public/production workflows.
+
+    Moreover, feedback on its implementation will be very appreciated.
+
+After you run a calculation in a remote machine, AiiDA will create a :py:class:`~aiida.orm.nodes.data.remote.RemoteData` node pointing at the folder where it was executed.
+However, this folder and all of its files remain in the remote machine.
+Some of these files might have been :ref:`retrieved<topics:calculations:usage:calcjobs:file_lists_retrieve>` when the calculation finished, but there will be no local copy for the rest of them.
+
+In order to give the users more flexibility when deciding what files to keep locally, AiiDA ships with a calculation plugin called :py:class:`~aiida.calculations.transfer.TransferCalculation`.
+You can use this feature to get files from a remote machine to be saved in a local :py:class:`~aiida.orm.nodes.data.folder.FolderData`.
+The specifications of what to copy are provided through an input of type
+
+.. code-block:: ipython
+
+    In [1]: instructions_cont = {}
+        ... instructions_cont['retrieve_files'] = True
+        ... instructions_cont['symlink_files'] = [
+        ...     ('node_keyname', 'source/path/filename', 'target/path/filename'),
+        ... ]
+        ... instructions_node = orm.Dict( dict = instructions_cont )
+
+The ``'source/path/filename'`` and ``'target/path/filename'`` are both relative paths (to their respective folders).
+The ``node_keyname`` is a string that will be used when providing the source :py:class:`~aiida.orm.nodes.data.remote.RemoteData` node to the calculation.
+You also need to provide the computer between which the transfer will ocurr:
+
+.. code-block:: ipython
+
+    In [2]: transfer_builder = CalculationFactory('core.transfer').get_builder()
+        ... transfer_builder.instructions = instructions_node
+        ... transfer_builder.source_nodes = {'node_keyname': source_node}
+        ... transfer_builder.metadata.computer = computer_node
```

Comment:
a `Computer` instance is not really a node. It doesn't live in the node table and also is not a subclass of `Node`. You also don't explain how to get it.
```suggestion
        ... transfer_builder.metadata.computer = source_node.computer
```

Reply:
So, the thing is that this fix works for when the `source_node` is of type `RemoteData` (which is the case for the first example). However, in the second example I say they can do the same thing but just changing the `dict` setup, so if they copy this for the second part it won't work. There they have to load the remote computer they want to use and pre-set it themselves, I cannot give further details of which one they want to use. I can remove the `_node` part though.

Reply:
I would actually keep the change I suggested and then simply explain what you just did in the example with the `FolderData` source node. There I would then suggest to show them to use `load_computer('some-computer-label')`. This is the typical way to load a `Computer` instance through the API. You can make it as explicit as the following for clarity
```
transfer_builder.metadata.computer = load_computer('some-computer-label')
```